plaso/engine/path_helper.py

```diff
@@ -120,7 +120,7 @@ def ExpandUsersHomeDirectoryPath(cls, path, path_separator, user_accounts):
       user_paths = []
       for user_account in user_accounts:
         user_path = user_account.user_directory
-        # Prevent concatting 2 path segment separators.
+        # Prevent concatenating 2 path segment separators.
```

Comment:
I think what we are trying to prevent here is not concatenating 2 path segment separators, but 2 successive path segment separators.

Reply:
Done.